.github/workflows/release.yml

```diff
@@ -14,3 +14,4 @@ jobs:
     secrets:
       github-token: ${{ secrets.BOT_TOKEN }}
       npm-token: ${{ secrets.NPM_BOT_TOKEN }}
+      
```

Comment:
```suggestion

```

Again, there should be a new line, but no space.

Reply:
okey ,done